### PR TITLE
DOC:Add examples for integrate.quadpack docstrings

### PR DESCRIPTION
**Pull request by samyak0210**
fixed https://github.com/scipy/scipy/issues/7168

### CODE REVIEW

**Comment by ilayn** [comment]
@samyak0210  Thank you for these docstrings. You are not actually solving the whole issue so you don't need to put fixes #7168 to these PRs.

**Comment by samyak0210** [comment]
@ilayn thank you for telling me. I am new to open source hence I didn't knew about that

**Comment by samyak0210** [comment]
@ilayn can you review the changes done.

**Comment by samyak0210** [comment]
@ev-br no one was replying so I thought you could review the changes.

**Comment by samyak0210** [comment]
@ev-br no one was replying so I thought you could review the changes.

**Comment by person142** [comment]
@samyak0210 sometimes things move slowly around here (everyone is volunteering their limited free time and so on), so it would be nice to only ping at a rate of once per week or so. :-)

File: scipy/integrate/quadpack.py [hunk 1]
```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
...
     scipy.special: For coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Returns triple integration of ``x*y*z``
+    x ranging from 1 to 2,y ranging from 2 to 3,z ranging from 0 to 1
+
+    >>> from scipy import integrate
+    >>> f=lambda z,y,x: x*y*z
```
Comment (andyfaff) [inline scipy/integrate/quadpack.py]: PEP8 space on either side of equals sign.

File: scipy/integrate/quadpack.py [hunk 2]
```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
     odeint: ODE integrators
     scipy.special: For coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Returns triple integration of ``x*y*z``
```
Comment (andyfaff) [inline scipy/integrate/quadpack.py]: --> Triple integration of ``x * y * z``

File: scipy/integrate/quadpack.py [hunk 3]
```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
...
+    x ranging from 1 to 2,y ranging from 2 to 3,z ranging from 0 to 1
```
Comment (andyfaff) [inline scipy/integrate/quadpack.py]: `x` ranging from 1 to 2, `y` ranging from ...
Spaces after comma and variables in single backticks.

File: scipy/integrate/quadpack.py [hunk 4]
```diff
@@ -633,6 +633,18 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
...
+
+    >>> from scipy import integrate
+    >>> f=lambda z,y,x: x*y*z
+    >>> integrate.tplquad(f,1,2,lambda x: 2,lambda x: 3,lambda x,y: 0,lambda x,y: 1)
```
Comment (andyfaff) [inline scipy/integrate/quadpack.py]: spaces after commas. If line goes over 80 chars, then carry over to next line (no \ required)

File: scipy/integrate/quadpack.py [hunk 5]
```diff
@@ -636,12 +636,13 @@ def tplquad(func, a, b, gfun, hfun, qfun, rfun, args=(), epsabs=1.49e-8,
...
 
-    Returns triple integration of ``x*y*z``
-    x ranging from 1 to 2,y ranging from 2 to 3,z ranging from 0 to 1
+    Returns triple integration of ``x * y * z``
+    x ranging from 1 to 2, y ranging from 2 to 3, z ranging from 0 to 1
 
     >>> from scipy import integrate
-    >>> f=lambda z,y,x: x*y*z
-    >>> integrate.tplquad(f,1,2,lambda x: 2,lambda x: 3,lambda x,y: 0,lambda x,y: 1)
+    >>> f = lambda z,y,x: x*y*z
```
Comment (ilayn) [inline scipy/integrate/quadpack.py]: still needs whitespace after commas

File: scipy/integrate/quadpack.py [hunk 6]
```diff
@@ -563,13 +563,13 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
...
 
-    Returns double integration of ``x*y**2``
+    Returns double integration of ``x * y**2``
     Integrating over rectangle with x ranging from 0 to 2
     and y ranging from 0 to 1
 
     >>> from scipy import integrate
-    >>> f=lambda y,x: x*y**2
-    >>> integrate.dblquad(f,0,2,lambda x: 0,lambda x: 1)
+    >>> f = lambda y,x: x*y**2
```
Comment (ilayn) [inline scipy/integrate/quadpack.py]: still needs whitespace after commas

**Comment by ilayn** [comment]
Thank you @samyak0210. For the future PRs about example docstrings please don't use "Fixed #7168" instead you can just write "cross ref: #7168 "

**Pull request merged**